modules/H2store/examples/ExampleH2BrineSolubility.m

```diff
@@ -0,0 +1,287 @@
+either version 3 of the License, or (at your option) any later version.
+
+MRST is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with MRST. If not, see <http://www.gnu.org/licenses/>.
+%}
+
+% Input Parameters for Temperature, Pressure, and Salinity
+min_temp     = 1;                 % [°C]
+max_temp     = 99;                % [°C]
+min_pressure = 6 * mega * Pascal; % [Pa]
+max_pressure = 20 * mega * Pascal;% [Pa]
+nbp          = 15;                % Number of pressure points
+nbt          = 15;                % Number of temperature points
+ms           = 0;                 % Salt molality [mol/kg]
+
+
+%% Notice on Computational Cost
+warning('ComputationalCost:Medium', ...
+       ['Please be advised that for large nbp and nbt this example often takes a long time ', ...
+        'to run: this script will extract datan from https://webbook.nist.gov']);
+% Get directory of current script
+currentDir = fileparts(mfilename('fullpath'));
+
+% Define target output directory and create if it doesn't exist
+output_dir = fullfile(currentDir, 'PVT', 'H2SolubilityTable');
+if ~exist(output_dir, 'dir')
+    mkdir(output_dir);
+end
+
+% Display options
+outputDisplay = false; % Set to true to display generated tables
+
+% This configuration prepares solubility data for the H2-brine system
+% under RK-EOS in the context of saline aquifer storage.
+
+% Generate H2O pure Component Table from NIST
+comp_name = 'H2O';
+disp(['Generating component table for: ', comp_name]);
+tab_H2O = generateComponentProperties('min_temp',min_temp, 'max_temp',max_temp, 'n_temp', nbt, 'min_press',min_pressure, 'max_press', max_pressure, 'n_press',nbp, 'comp_name', comp_name,'display_output', true,'output_dir',output_dir);
+% Generate H2 pure Component Table from NIST
+pause(0.1);  % Ensure smooth execution between commands
+comp_name = 'H2';
+disp(['Generating component table for: ', comp_name]);
+tab_H2 = generateComponentProperties('min_temp',min_temp, 'max_temp',max_temp, 'n_temp', nbt, 'min_press',min_pressure, 'max_press', max_pressure, 'n_press',nbp, 'comp_name', comp_name,'display_output', true,'output_dir',output_dir);
+% We use the Redlick Kwong Eos to obtain the solubility
+disp('Generating solubility table for H2-brine mixture...');
+%[tab_sol, status_sol, file_path_sol] = generateSolubilityTable(min_temp, max_temp, min_pressure, max_pressure, nbp, nbt, ms, output_dir);
+tab_sol= generateH2WaterSolubilityTable('min_temp',min_temp, 'max_temp',max_temp, 'n_temp', nbt,'min_press',min_pressure, 'max_press',max_pressure, 'n_press', nbp, 'ms', ms,'display_output', true,'output_dir',output_dir);
+% Load ePC-SAFT Data
+epcsaft = load('./thermodymics/PcSaftSolubilityTable/ePcSaftH2BrineData.mat');
```

Comment:
Relative paths don't work in this way as the user may well be running the script whilst in a different working directory. 

Instead you can use the mrst variable ROOTDIR which gives the location to the base MRST folder. 

```suggestion
epcsaft = load(fullfile(ROOTDIR,'..','modules','H2store','thermodynamics',...
    'PcSaftSolubilityTable','ePcSaftH2BrineData.mat'));
```
This should be done for all paths.
However, this particular file does not exist at this location anyway!
Can you go through *all* examples and check that all paths are correct.